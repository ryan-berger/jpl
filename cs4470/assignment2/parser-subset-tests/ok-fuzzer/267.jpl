show O
write image o(.228234, 3973, 00.) to ""
time show H
write image .8317877 to "G"
print "wU"
