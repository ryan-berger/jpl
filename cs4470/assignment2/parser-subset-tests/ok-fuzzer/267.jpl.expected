(ShowCmd (VarExpr O))
(WriteImageCmd (CallExpr o (FloatExpr 0) (IntExpr 3973) (FloatExpr 0)) "")
(TimeCmd (ShowCmd (VarExpr H)))
(WriteImageCmd (FloatExpr 0) "G")
(PrintCmd "wU")
